File: OOP practice/constructor.py
class Employee:
    def __init__(self,name,company,age):
        self.name=name
        self.company=company
        self.age=age

if __name__ == "__main__":
    list1=[];
    no_of_emp=int(input())
    for i in range(no_of_emp):
        name,company=input()
        age=int(input())
        list1.append(Employee(name,company,age))
    list1.append(Employee("aniket","Google",19))
    list1.append(Employee("Rahul","Microsoft",19))
    list1.append(Employee("Prakash","Adobe",29))
    list1.append(Employee("Sumit","Amazon",23))
    list1.append(Employee("Praniket","Netflix",21))    
    
    for i in list1:
        print("{}-{}-{}".format(i.name,i.company,i.age))
        

File: OOP practice/constructor.cpp
#include<iostream>
using std::string;
class Employee
{
    private:
        std::string name;
        std::string company;
        int age;
    public:

        void setter(string Name,string Company,int Age)
        {
            name=Name;
            company=Company;
            age=Age;
        }

        string gettername()
        {
            return name;
        }

         string gettercompany()
        {
            return company;
        }

         int getterage()
        {
            return age;
        }

        Employee(std::string Name,std::string Company,int Age)
        {
            name=Name;
            company=Company;
            age=Age;
        }
};


int main()
{
    // Employee employee1=Employee("aniket","Google",19);
    
    int n;
    cin>>n;
    Employee employees[n];
    for(int i=0;i<n;i++)
    {
        string name;
        string company;
        int age;
        employees[i].setter(name,company,age);
    }
    for(int val:range(n))
    {
        std::cout<<val.name<<val.company<<val.age<<std::endl;
    }
    return 0;
}